src/Controller.js

```diff
@@ -41,6 +41,10 @@ class Controller {
     return this.itemStore.isDirty();
   }
 
+  errors() {
```

Comment:
hasErrors to be consistent and indicate this is a boolean, not returning the set of errors

Reply:
I did think that too, I was trying to keep it consistent with the function above it which goes from isDirty() in the itemStore to dirty() in the controller
Should I change that one also?

Reply:
Oops.  No, I think just change your one.  I'm doing some refactoring at the moment and will try to remember to tidy dirty()